README.md

```diff
@@ -31,6 +31,59 @@ LEMMY_BB_BACKEND=https://example.com cargo run
 
 Then open [127.0.0.1:1244](http://127.0.0.1:1244) in your browser. You can login with your existing account.
 
+### Standalone
+
+Follow these steps to install only the lemmyBB frontend on your server.
+
+First, ssh into your server and prepare by cloning the code repository.
+```
+cd /opt
+git clone https://github.com/LemmyNet/lemmyBB.git
+```
+
+Change to the folder, create lemmybb binary and create a link of it to the root folder
+
+```
+cd lemmyBB
+cargo build --release
+ls /opt/lemmyBB/target/debug/lemmy_bb /opt/lemmyBB/lemmy_bb
+```
+
+Copy the nginx config into the sites-enabled folder and edit it
+```
+cp docker/nginx-lemmybb.conf /etc/nginx/sites-enabled/lemmybb.conf
+```
+
+create systemd service file
+```
+nano /etc/systemd/system/lemmy_bb.service
+```
+
+and insert the following content and adapt 'LEMMY_BB_BACKEND' and 'LEMMY_BB_LISTEN_ADDRESS' to your installation
+```
+[Unit]
+Description=lemmy_bb
+After=network.target
+
+[Service]
+Type=simple
+WorkingDirectory=/opt/lemmyBB/
+Environment="LEMMY_BB_BACKEND=http://127.0.0.1:8536"
+Environment="LEMMY_BB_LISTEN_ADDRESS=127.0.0.1:8703"
+Environment="LD_PRELOAD=libjemalloc.so"
+ExecStart=/opt/lemmyBB/lemmy_bb
+Restart=always
+
+[Install]
+WantedBy=multi-user.target
+```
+
+then activate and start the service and check the status
+```
+systemctl enable --now lemmy_bb.service 
+systemctl status lemmy_bb.service
+```
+
 ### Installation
```

Comment:
This should be like "Installation together with new Lemmy instance", or similar.